servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -3794,7 +3794,7 @@ Policy getPolicy(ObjectStoreConnection con, String domainName, String policyName
         Policy policy = con.getPolicy(domainName, policyName, version);
         if (policy != null) {
             policy.setAssertions(con.listAssertions(domainName, policyName, version));
-            policy.setTags(con.getPolicyTags(domainName, policyName, version));
+            policy.setTags(con.getPolicyTags(domainName, policyName, version != null ? version : policy.getVersion()));
```

Comment:
This change should not be necessary. In the last PR I fixed the logic to handle versions correctly. So please verify that your test case works without this change.